apps/desktop-wallet/src/modals/SecretPhraseModal.tsx

```diff
@@ -26,18 +27,37 @@ import { Section } from '@/components/PageComponents/PageContainers'
 import PasswordConfirmation from '@/components/PasswordConfirmation'
 import { useAppSelector } from '@/hooks/redux'
 import CenteredModal from '@/modals/CenteredModal'
+import { walletStorage } from '@/storage/wallets/walletPersistentStorage'
 
 const SecretPhraseModal = ({ onClose }: { onClose: () => void }) => {
   const { t } = useTranslation()
-  const activeWalletMnemonic = useAppSelector((state) => state.activeWallet.mnemonic)
+  const activeWalletId = useAppSelector((s) => s.activeWallet.id)
   const [isDisplayingPhrase, setIsDisplayingPhrase] = useState(false)
+  const [mnemonic, setMnemonic] = useState<string>()
```

Comment:
I lied, this is the second place where we cache the mnemonic. We need to display it in the UI somehow.

Reply:
Again, inevitable.